rest/config.go

```diff
@@ -830,6 +830,18 @@ func (sc *StartupConfig) validate() (errorMessages error) {
 	if sc.Bootstrap.Server == "" {
 		errorMessages = multierror.Append(errorMessages, fmt.Errorf("a server must be provided in the Bootstrap configuration"))
 	}
+
+	secureServer := base.ServerIsTLS(sc.Bootstrap.Server)
+	if base.BoolDefault(sc.Bootstrap.UseTLSServer, true) {
```

Comment:
I guess we should have a `DefaultUseTLSServer` constant for this value if we need to know it in more than just `DefaultStartupConfig()`